chunky/src/java/se/llbit/chunky/renderer/RenderContext.java

```diff
@@ -66,47 +77,55 @@ public int numRenderThreads() {
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
    * @return Scene description file
    */
-  public File getSceneDescriptionFile(String sceneName) {
-    return getSceneFile(sceneName + Scene.EXTENSION);
+  public File getSceneDescriptionFile(String descriptionFileName) {
+    return getSceneFile(descriptionFileName + Scene.EXTENSION);
   }
 
   /**
+   * @param descriptionFileName The name of the scene description file. NOTE: Do not include extension
    * @return Input stream for the scene description
-   * @throws FileNotFoundException
+   * @throws FileNotFoundException If the file does not exist.
    */
-  public InputStream getSceneDescriptionInputStream(String sceneName) throws FileNotFoundException {
-    return getSceneFileInputStream(sceneName + Scene.EXTENSION);
+  public InputStream getSceneDescriptionInputStream(String descriptionFileName) throws FileNotFoundException {
```

Comment:
:thinking: If the "descriptionFileName" must not contain the extension, we should keep calling it `sceneName`; same for the case above